kubernetes/charts/direktiv/templates/prometheus-rolebind-ns.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+ name: {{ include "direktiv.serviceAccountName" . }}-prometheus-binding-lock
```

Comment:
I called it something with "-lock" because it was actually permissions for a lock.

Reply:
replaced `-lock` with `-ns`, as it's a namespace-scoped RoleBinding